### ISSUE TITLE
Is somebody still working on this?

### ISSUE THREAD

**Issue by kaimast** [jbenet/bson-cpp#5]
It seems like libbson is in pretty good shape but bson-cpp is apparently abandoned? Is there another, more up to date, repository?

**Comment by jbenet** [comment]
Hey @kaimast sorry I no longer maintain this. Happy to merge any PRs though

**Comment by kaimast** [comment]
Cool thanks. It feels like it might make more sense to have a thin c++ layer on top libbson though, instead of re-implementing everything in C++.

**Comment by jbenet** [comment]
@kaimast yeah absolutely, when I made this that didnt exist. I ripped this out of the mongodb driver

**Comment by dwight** [comment]
hi, a few comments.  thanks for taking the time to make/package this previously.

first, i was unaware of this project and did something very similar not too long ago -- see https://github.com/dwight/bson-cxx.  it's a little different, see the readme for details.

second, there is a question about the license in the README of this project.  the BSON code from the MongoDB C++ Driver should all be Apache 2.0.  So you should be able to use that for everything herein, assuming that is all it is using (i haven't read everything herein).

**Comment by jbenet** [comment]
Hello @dwight! I just granted you contributor access to this project. Do what you want with it. (like deprecate it and point people towards yours, or fix the license wording to satisfy the constraints). :smile:

**Comment by dwight** [comment]
thanks
